### ISSUE TITLE
Missing OpenEXR CMake module

### ISSUE THREAD

**Issue by charlesfleche** [MercenariesEngineering/openexrid#7]
CMake build fails with missing package `FindOpenEXR.cmake`:

```
CMake Error at openexrid/CMakeLists.txt:1 (FIND_PACKAGE):
  By not providing "FindOpenEXR.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "OpenEXR", but
  CMake did not find one.

  Could not find a package configuration file provided by "OpenEXR" with any
  of the following names:

    OpenEXRConfig.cmake
    openexr-config.cmake

  Add the installation prefix of "OpenEXR" to CMAKE_PREFIX_PATH or set
  "OpenEXR_DIR" to a directory containing one of the above files.  If
  "OpenEXR" provides a separate development package or SDK, be sure it has
  been installed.
```

I've the feeling that either `OpenEXRConfig.cmake` or `openexr-config.cmake` is installed locally on Mercenaries workstations, but not released in `openexrid`. What `OpenEXRConfig.cmake` are you using ? Is that the one from USD ?

**Comment by 1LOVESJohnny** [comment]
> CMake build fails with missing package `FindOpenEXR.cmake`:
> 
> ```
> CMake Error at openexrid/CMakeLists.txt:1 (FIND_PACKAGE):
>   By not providing "FindOpenEXR.cmake" in CMAKE_MODULE_PATH this project has
>   asked CMake to find a package configuration file provided by "OpenEXR", but
>   CMake did not find one.
> 
>   Could not find a package configuration file provided by "OpenEXR" with any
>   of the following names:
> 
>     OpenEXRConfig.cmake
>     openexr-config.cmake
> 
>   Add the installation prefix of "OpenEXR" to CMAKE_PREFIX_PATH or set
>   "OpenEXR_DIR" to a directory containing one of the above files.  If
>   "OpenEXR" provides a separate development package or SDK, be sure it has
>   been installed.
> ```
> 
> I've the feeling that either `OpenEXRConfig.cmake` or `openexr-config.cmake` is installed locally on Mercenaries workstations, but not released in `openexrid`. What `OpenEXRConfig.cmake` are you using ? Is that the one from USD ?

Hi @charlesfleche, I met the same problem. Have you solved it?
Expect for your reply.

**Comment by charlesfleche** [comment]
No, I just tried it again, still no success. I suspect that the conan build works because it somehow pulls the `FindIlmBase.cmake` and `FindOpenEXR.cmake` files.
